src/package.json

```diff
@@ -53,7 +53,7 @@
   },
   "homepage": "https://uxaspects.github.io/UXAspects",
   "dependencies": {
-    "angular-split": "1.0.0-rc.3",
+    "angular-split": "3.0.2",
```

Comment:
This should probably be ^3.0.2 to allow some flexibility. We can be reasonably sure that 3.x updates will not contain breaking changes.

Reply:
I think this comment was missed.